_studio/shared/include/libmfx_core_vaapi.h

```diff
@@ -209,6 +209,8 @@ class VAAPIVideoCORE : public CommonCORE
 #ifdef MFX_ENABLE_MFE
     ComPtrCore<MFEVAAPIEncoder> m_mfe;
 #endif
+    //required to WA FEI enabling after move it from plugin to library
```

Comment:
I don't like idea of having just HEVC FEI specific member in the core. I'd like to see more generic interface.

Reply:
I'd also like, but we don't have generic usage for this.